frb_codegen/src/lib.rs

```diff
@@ -111,11 +111,20 @@ pub fn frb_codegen(config: &config::Opts, all_symbols: &[String]) -> anyhow::Res
         EXTRA_EXTERN_FUNC_NAMES.to_vec(),
     ]
     .concat();
+
     let c_dummy_code = generator::c::generate_dummy(&effective_func_names);
     for output in &config.c_output_path {
         fs::create_dir_all(Path::new(output).parent().unwrap())?;
+        let output_new: String = if config.count > 1 {
+            output
+                .clone()
+                .replace(".h", format!("_{}.h", config.idx).as_str())
```

Comment:
Hmm I am not sure whether this is the correct way to do so. In order to be consistent with the remaining part, maybe we should let the user input multiple paths (i.e. a vec of paths)?

Reply:
In the non-multi files scenario, we will set the path to ios and macos.
In addition, these paths represent the same result.
As before, each path shows only the last header file